providers/unix/components/process/process.go

```diff
@@ -51,6 +50,11 @@ func (p *Process) start(ctx context.Context) error {
 		gracePeriod = *p.ShutdownGracePeriodSeconds
 	}
 
+	childEnv, err := json.Marshal(p.Environment)
```

Comment:
Yeah, more JSON strings :)

Reply:
I reassigned https://github.com/deref/exo/issues/47 to you. Please improve :)